src/vmm_mad/remotes/lib/lxd/opennebula_vm.rb

```diff
@@ -207,7 +208,7 @@ def storage(hash)
         disks = @xml.elements('//TEMPLATE/DISK')
 
         disks.each do |n|
-            next unless valid_disk?(n)
+            next if swap?(n)
```

Comment:
See comments below, probably valid_disk is still needed

Reply:
valid_disk? was replaced by swap, because it stood for unsupported disks type , which were volatile fs and swap files. Now only swap.